databaseserviceexample.gitlab-ci.yml

```diff
@@ -0,0 +1,53 @@
+include: 
+    # Basic node service ci core 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/belt-node-complexity-analysis-ci.yml
+    # Reusable components for deployment and/or data migration. 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/deploycomponents.yaml
+    # Will automatically run setup scripts for building, testing and deploying node based software and services.
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/setup/nodesetup.yaml
+    # Includes the basic stages of a database service build. 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/database/stages.yaml
+    # Allows for the use of .buildDatabaseWithNpm to extend your build with database support 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/database/builddatabase.yaml
+    # Sets up a pipeline targeted for elastic beanstalk deployment. 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/beanstalkdeploy.yaml
+    # Will scan your dependencies for volunerabilities. 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/service/dependencyscanning.yaml
+
+build:myDatabaseService:
+  extends: .testNodeApplicationWithDatabase
+  variables:
+    POSTGRES_VERSION: postgres:10.6-alpine
+    POSTGRES_DB: myDbName
+    POSTGRES_USER: myDBUser
+    POSTGRES_SERVER: MyDbService
+    POSTGRES_PASSWORD: ''
+
+migrate-int:
+  extends: .migrate
+  stage: migrate-int
+  only:
+    - master
+  except:
+    - schedules
+  variables:
+    CI_ENVIRONMENT_NAME: integration
+    POSTGRES_SERVER: myDatabaseServerAddress.com
+    POSTGRES_USER: myDBUser
+    POSTGRES_DB: MyDbService
+    POSTGRES_PASSWORD: $INT_DB_PASSWORD # $INT_DB_PASSWORD needs to be set in git lab varables 
+
+migrate-prd:
+  extends: .migrate
+  stage: migrate-prd
+  when: manual
+  only:
+    - master
+  except:
+    - schedules
+  variables:
+    CI_ENVIRONMENT_NAME: production
```

Comment:
In conjunction with https://github.com/Cimpress-MCP/gitlab-continuous-integration/pull/22#discussion_r254124140, this probably needs to be set to "prd" (and integration needs to be set to "int").